sdk/core/Azure.Core/samples/Serialization.md

```diff
@@ -134,12 +165,10 @@ options.GenericTypeSerializerCreator = type => type.Equals(typeof(ModelT)) ? new
 Envelope<ModelT> model = ModelSerializer.Deserialize<Envelope<ModelT>>(new BinaryData(Encoding.UTF8.GetBytes(serviceResponse)), options: options);
 ```
 
-## Using ModelSerializer with generic IModelSerializable
+## Interfacing with the low level interfaces
```

Comment:
```suggestion
## Interfacing with the low-level interfaces
```

I'd suggest that we refine this title.   "Low-level interfaces" is a subjective term and doesn't really tell me what they are - just what we call them internally.   To be honest, I wanted to make suggestions here for alternatives, but I have no idea what the heck these are.